From 18b733fb082bb98ba37d16e5b184f6d959d0f573 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Thu, 12 Dec 2024 20:02:38 +0100
Subject: [PATCH] Fix fmt includes again

Unfortunately, VTK upstream imports new releases and
creates the same regressions again and again ...
---
 ThirdParty/ioss/vtkioss/Ioss_Field.C        | 1 +
 ThirdParty/ioss/vtkioss/Ioss_VariableType.C | 1 +
 2 files changed, 2 insertions(+)

diff --git a/ThirdParty/ioss/vtkioss/Ioss_Field.C b/ThirdParty/ioss/vtkioss/Ioss_Field.C
index fbb2100c..8909eef5 100644
--- a/ThirdParty/ioss/vtkioss/Ioss_Field.C
+++ b/ThirdParty/ioss/vtkioss/Ioss_Field.C
@@ -12,6 +12,7 @@
 #include <cstdint>
 #include "vtk_fmt.h"
 #include VTK_FMT(fmt/ostream.h)
+#include VTK_FMT(fmt/ranges.h)
 #include <iostream>
 #include <string>
 #include <vector>
diff --git a/ThirdParty/ioss/vtkioss/Ioss_VariableType.C b/ThirdParty/ioss/vtkioss/Ioss_VariableType.C
index b081dc36..4f1a841f 100644
--- a/ThirdParty/ioss/vtkioss/Ioss_VariableType.C
+++ b/ThirdParty/ioss/vtkioss/Ioss_VariableType.C
@@ -19,6 +19,7 @@
 #include VTK_FMT(fmt/core.h)
 #include VTK_FMT(fmt/format.h)
 #include VTK_FMT(fmt/ostream.h)
+#include VTK_FMT(fmt/ranges.h)
 #include <map>
 #include <sstream>
 #include <string>
-- 
2.47.1

